packages/flutter/lib/src/gestures/recognizer.dart

```diff
@@ -151,7 +151,19 @@ abstract class GestureRecognizer extends GestureArenaMember with DiagnosticableT
 /// simultaneous touches to each result in a separate tap.
 abstract class OneSequenceGestureRecognizer extends GestureRecognizer {
   /// Initialize the object.
-  OneSequenceGestureRecognizer({ Object debugOwner }) : super(debugOwner: debugOwner);
+  ///
+  /// It's possible to limit this recognizer to a specific [PointerDeviceKind]
```

Comment:
You should turn this text into a dartdoc macro.  Add this before it:
```dart
/// {@template flutter.gestures.oneSequenceGestureRecognizer.kind}
```
and this after:
```dart
/// {@endtemplate}
```

And then, in all the other places you have this text, just replace it with:
```dart
/// {@macro flutter.gestures.oneSequenceGestureRecognizer.kind}
```